src/app/modules/course-overview/components/course-overview/course-overview.component.scss

```diff
@@ -0,0 +1,35 @@
+
+@import './src/styles.scss';
+
+:host {
+  box-sizing: border-box;
+}
+
+h1 {
+  @extend .frank;
+  font-size: 72pt;
```

Comment:
use vw?

Reply:
I'm not sure if this is necessary.

Since I will be changing the font sizes in the lessons as well, I will change this in that PR as well. I'm leaning towards media queries over vw or maybe a combination of the two.